core/src/main/java/io/questdb/cairo/TableNameRegistryRW.java

```diff
@@ -59,6 +59,11 @@ public boolean dropTable(TableToken token) {
         return false;
     }
 
+    @Override
+    public boolean isTableNameLocked(String tableName) {
+        return nameTableTokenMap.containsKey(tableName);
```

Comment:
nit: technically this check doesn't do what the method name suggests. A table name is locked only when the map contains a `<tableName, LOCKED_TOKEN>` pair. To make things slightly more obvious either the check could be changed, or the method could be renamed to `isTableNameAvailable()`.

Reply:
I want to see if leaving it as it was has an effect on the concurrent tests, and if not, then I will revert this change, makes things less readable, and for no much gain.

Reply:
The gain is definitely questionable considering that we're talking of DDLs on a particular table.